package.json

```diff
@@ -47,7 +47,7 @@
     "test:update": "yarn run test --updateSnapshot",
     "test": "LC_ALL=en_US.UTF-8 jest",
     "test:unit": "yarn test --testPathIgnorePatterns='src/__tests__/a11y.test.tsx'",
-    "test:a11y": "yarn test src/__tests__/a11y.test.tsx",
+    "test:a11y": "yarn test src/__tests__/a11y.test.tsx --testPathIgnorePatterns='src/components' ",
```

Comment:
Is `testPathIgnorePatterns` mandatory here?

Reply:
Yes I tried many different solutions and that's the only way I found. Let me explain:

if you do `yarn test:a11y src/components/Alert` without this parameter jest will launch `a11y.test.tsx` but also `src/components/Alert/__tests__/index.test.tsx`.

while we only want to give parameter to precise which a11y test should be run we also trigger normal unit tests.

Reply:
Could we instead have two jest config files ? 

This broke jest test narrowing a while ago :)

```bash
$ yarn run test:unit src/component/Alert # this used to only start Alert tests, now it runs everything
```

Reply:
@chambo-e I didn't saw that issue, I did split configuration file in two :)